pkg/flare/archive_docker_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build docker
+
+package flare
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTrimCommand(t *testing.T) {
+	for in, out := range map[string]string{
+		"/pause":                                     "/pause",
+		"nginx -g 'daemon off;'":                     "nginx …",
+		"/entrypoint.sh datadog-cluster-agent start": "/entrypoint.sh …",
+		"/coredns -conf /etc/coredns/Corefile":       "/coredns …",
+		"/my/very/long/command":                      "/my/very/long/comm…",
```

Comment:
this does not look that long to me, should we increase it if we only keep the command? 🤔

Reply:
Increasing this would break the column layout as we use fixed 20 char wide columns
https://github.com/DataDog/datadog-agent/pull/2813/files#diff-2d9a96468dd1400f6fe96a4223974ee5R93 
As the tabwriter we use does not support different width per column, we'd need to either tabulate with another logic, or increase the global width. WDYT?

Reply:
`20` is the minimum column width, the maximum width is elastic, on examples I checked the image + names columns are very long (120+ chars)

Reply:
Ah, cool! Shall we go with 29 then (30 with the `...`) then?